http/src/main/resources/reference.conf

```diff
@@ -964,7 +964,7 @@ terra-app-setup-chart {
   # cert files into the local chart.
   chart-name = "/leonardo/terra-app-setup"
   # If you change this here, be sure to update it in the dockerfile
```

Comment:
Is this docstring incorrect, outdated? Or does a value in a `Dockerfile` need to be bumped?